docs/src/main/sphinx/release/release-456.md

```diff
@@ -0,0 +1,86 @@
+# Release 456 (5 Sep 2024)
+
+## General
+
+* Expose additional JMX metrics about resource groups, including CPU and memory
+  usage, limits, and scheduling policy. ({issue}`22957`)
+* Add [adaptive plan optimization](/optimizer/adaptive-plan-optimizations) which
+  can reorder joins at runtime. The feature can be disabled with the
+  `fault-tolerant-execution-adaptive-join-reordering-enabled` configuration
+  property or the `fault_tolerant_execution_adaptive_join_reordering_enabled`
+  session property. ({issue}`23046`)
+* Improve performance for [file compression and decompression](file-compression)
+  of ORC and Parquet files used for [exchange spooling with fault-tolerant
+  execution mode](fte-exchange-manager). ({issue}`22532`)
+* Extend the Kafka event listener to send split completion events. ({issue}`23065`)
+
+## JDBC driver
+
+* Publishing a JDBC driver JAR without bundled, third-party dependencies. ({issue}`22098`)
+
+## BigQuery connector
+
+* Fix failures with queries using table functions when `parent-project-id` is
+  defined. ({issue}`23041`)
+
+## Blackhole connector
+
+* Add support for replacing tables. ({issue}`23004`)
```

Comment:
Might be better to say `CREATE OR REPLACE TABLE` support.

Reply:
Update to same wording we used in Delta Lake in the past